src/workflows-app/RunDetails.js

```diff
@@ -279,24 +268,10 @@ export const BaseRunDetails = (
           ]),
       ],
       () =>
-        div([
+        div({ style: { width: '100%' } }, [
           div({ style: { padding: '1rem 2rem 2rem' } }, [header]),
           div({ style: { display: 'flex', justifyContent: 'space-between', padding: '1rem 2rem 1rem' } }, [
-            h(WorkflowInfoBox, { workflow }, []),
-            h(
-              TroubleshootingBox,
-              {
-                name,
-                namespace,
-                logUri: workflow.workflowLog,
-                submissionId,
-                workflowId,
-                showLogModal,
-                appId: appIdMatched,
-                workflowName: getWorkflowName(),
-              },
-              []
-            ),
+            h(WorkflowInfoBox, { workflow, name, namespace, submissionId, workflowId, workspaceId, showLogModal }, []),
```

Comment:
Nit: any empty children list can be omitted.

```suggestion
            h(WorkflowInfoBox, { workflow, name, namespace, submissionId, workflowId, workspaceId, showLogModal }),
```